content/en/docs/instrumentation/js/exporters.md

```diff
@@ -67,6 +67,109 @@ $ docker run -d --name jaeger \
   jaegertracing/all-in-one:latest
 ```
 
+### Usage with the WebTracer
+
+When you use the OTLP exporter in a browser-based application, you need to note
+that:
+
+1. grpc-based exporting is not supported
+2. [Content Security Policies][] (CSPs) of your website might block your exports
+3. [Cross-Origin Resource Sharing][] (CORS) headers might not allow your exports
+   being send
+4. You might need to expose your collector to the public internet
+
+Below you will find instructions to use the right exporter, to configure your
+CSPs and CORS headers and what precautions you have to take when exposing
+your collector.
+
+#### Use OTLP exporter with HTTP
+
+[OpenTelemetry Collector Exporter with grpc][] does only work with Node.JS,
+therefor you are limited to use the
+[OpenTelemetry Collector Exporter with http][]. This is not an issue, but you
```

Comment:
I believe this defaults to http/json and not http/proto. Is that worth mentioning?

Reply:
let me double check on that and update accordingly.